packages/ui/src/app/pages/WorkingGroups/MyApplications.tsx

```diff
@@ -18,20 +19,35 @@ export const MyApplications = () => {
     ],
     []
   )
-  const { applications } = useMyApplications()
+  const { applications, isLoading } = useMyApplications()
+  const isPending = (a: WorkingGroupApplication) => a.status == 'ApplicationStatusPending'
```

Comment:
Two things:

1.  Short names are cryptic the only good place for them is in for loops or in sort functions IMO. Having `application => application.status === '...'` is not that long :wink:. The other thing is destructuring should work: `( { status } ) => status === '...'`
2.  Re-creating helper functions on every render has its cost related to creating a new Function every time. This allocates memory, takes time, etc. This helper can be safely moved outside the component. This way it will be created once on module read.